wayfinder/README.md

```diff
@@ -0,0 +1,118 @@
+# Agent Check: Wayfinder
+
+## Overview
+
+[Wayfinder](https://docs.appvia.io/wayfinder) is an infrastructure management
+platform that enables developer self-service though centralised configuration.
+This check monitors Wayfinder key management components through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install the integration in Wayfinder Kubernetes management cluster.
+
+### Installation
+
+For containerized environments, the best way to use this integration with the Docker Agent is to build the Agent with the Wayfinder integration installed. 
+
+Pre-requisites:
+
+Network policy must be configured to allow the Datadog Agent to connect to Wayfinder components. Network policy below assumes Datadog is deployed to datadog namespace and Wayfinder is deployed to wayfinder namespace.
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: datadog-agent
+  namespace: wayfinder
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          name: datadog
+      podSelector:
+        matchLabels:
+          app: datadog-agent
+    ports:
+    - port: 9090
+      protocol: TCP
+  podSelector:
+    matchExpressions:
+    - key: name
+      operator: In
+      values:
+      - wayfinder-controllers
+      - wayfinder-apiserver
+      - wayfinder-webhooks
+  policyTypes:
+  - Ingress
+```
+
+To build an updated version of the Agent:
+
+1. Use the following Dockerfile:
+
+```dockerfile
```

Comment:
You may want to indent this code block by 3 spaces so that it's lined up under the numbered bullet. For #3 and #4 as well.